src/dev-app/youtube-player/youtube-player-demo.ts

```diff
@@ -6,7 +6,9 @@
  * found in the LICENSE file at https://angular.io/license
  */
 
- import {Component} from '@angular/core';
+import {
+  AfterViewInit, ChangeDetectorRef, Component, ElementRef, OnDestroy, ViewChild
```

Comment:
Should be split across multiple lines.

Reply:
Even if it fits on one line?

Reply:
I think that once we get to this many imports, we'll eventually have to split it across multiple lines anyway.

Reply:
OK, the only setting that I have in IntelliJ is to "chop down if too long".

I guess that I should just get use to running `yarn format:ts` as that does seem to do it properly. It requires that I stage the files first though, which is something that I don't normally have to deal with.